### PR TITLE
Cleanup AIAB reference.

### PR DESCRIPTION
**Pull request by thakurajayL**

### CODE REVIEW

File: developer/gnbsim.rst [hunk 1]
```diff
@@ -3,309 +3,13 @@
...
-* NW triggered PDU Session Release
-
-It is also capable to generate and send user data packets (ICMP echo request)
-and process down-link user data (ICMP echo response) over the established data
-plane path (N3 Tunnel).
-
-gNBSim Code References
-----------------------
+gNBSim is the default test tool used in the SD-Core Project (Private 5G project). gNBSim documentation is available
+in the gNBSim github repository
```
Comment (gab-arrobo) [inline developer/gnbsim.rst:11]:
```suggestion
in the gNBSim GitHub repository
```

File: developer/aiab5g.rst [hunk 2]
```diff
@@ -10,20 +10,3 @@ Aether In a Box - 5G
...
-
-    images:
-      tags:
-        webui: omecproject/5gc-webui:rel-1.4.0
-
-To upgrade a running 5G SD-CORE with the new image, or to redeploy the 5G
-SD-CORE with the image::
-
-    make reset-5g-test # delete 5G deployment if it was already started before updating image
-    make 5g-core  # now this deployment will use new webui image
```
Comment (gab-arrobo) [inline developer/aiab5g.rst:29]: I think we need to change this with `OnRamp` instead of removing it because it is similar to use as AiaB

Comment (thakurajayL) [reply developer/aiab5g.rst:29]: I do not want to keep partial instructions here. Its error prone. Updated description of this page and want to make sure developers visit onRamp page to get more help.

File: developer/aiab5g.rst [hunk 3]
```diff
@@ -7,23 +7,9 @@
 Aether In a Box - 5G
 ====================
 
-The documentation for Aether In a Box (AiaB) - 5G is located in `Aether OnRamp <https://docs.aetherproject.org/master/onramp/overview.html>`_,
-where specifics about setting up Aether 5G, running 5G test, troubleshooting,
-and other details are described.
+Previously developers used Aether in a Box (AIAB) environment to test SD-Core. But AIAB is
```
Comment (gab-arrobo) [inline developer/aiab5g.rst:10]:
```suggestion
Previously developers used Aether in a Box (AiaB) environment to test SD-Core. But AiaB is
```

**Review by gab-arrobo — APPROVED** [review]
+1

**Pull request merged**